### ISSUE TITLE
Update progress

### ISSUE THREAD

**Issue by Tyler-Hilbert** [Tyler-Hilbert/2024-Going-GitHub-Green-Competition#14]
@daishadavis do you want to update using these 4 images

https://web.archive.org/web/20240207220420/http://web.archive.org/screenshot/https://github.com/daishadavis?tab=overview&from=2024-02-01
![daishadavis](https://github.com/Tyler-Hilbert/2024-Going-GitHub-Green-Competition/assets/6968707/4f78d46c-f722-4331-b2a3-4887e074e700)

https://web.archive.org/web/20240207215702/http://web.archive.org/screenshot/https://github.com/DarrenBaldwin07?tab=overview&from=2024-02-01
![DarrenBaldwin07](https://github.com/Tyler-Hilbert/2024-Going-GitHub-Green-Competition/assets/6968707/26520b76-7c36-4476-8bd6-c595f2dfbca1)

https://web.archive.org/web/20240207215322/http://web.archive.org/screenshot/https://github.com/Jayko001?tab=overview&from=2024-02-01
![Jayko001](https://github.com/Tyler-Hilbert/2024-Going-GitHub-Green-Competition/assets/6968707/2ec18a31-d1df-41bd-919e-381ed1f55ef7)

https://web.archive.org/web/20240207214733/http://web.archive.org/screenshot/https://github.com/Tyler-Hilbert?tab=overview&from=2024-02-01
![Tyler-Hilbert](https://github.com/Tyler-Hilbert/2024-Going-GitHub-Green-Competition/assets/6968707/b427f40e-b95d-4459-a61a-716b72638f40)

**Comment by Tyler-Hilbert** [comment]
@daishadavis you want to update this? Or maybe find a way to automate it.

**Comment by daishadavis** [comment]
@Tyler-Hilbert  I'll update it but we should probably figure out a way to automate it

**Comment by Tyler-Hilbert** [comment]
Agree, can you create a separate issue for "automating data collection"
